container-engine-lib/lib/backend_interface/objects/image_download_mode/image_download_mode.go

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
in accord with the comment above, i propose we change this to reflect [service status](https://github.com/kurtosis-tech/kurtosis/blob/main/container-engine-lib/lib/backend_interface/objects/service/service_status.go)